src/handlers/web3.js

```diff
@@ -274,18 +304,22 @@ export const getTransactionCount = address =>
 export const getTxDetails = async transaction => {
```

Comment:
technically `getTxDetails` doesn't need to be exported as it is only used in this file